cdc/metadata.hh

```diff
@@ -54,8 +59,9 @@ public:
      *
      * If the provided timestamp is too far away "into the future" (where "now" is defined according to our local clock),
      * we reject the get_stream query. This is because the resulting stream might belong to a generation which we don't
-     * yet know about. The amount of leeway (how much "into the future" we allow `ts` to be) is defined
-     * by the `cdc::generation_leeway` constant.
+     * yet know about. Similarly, we reject queries to the previous generations if the timestamp is too far away "into
+     * the past" as it could break the consistency properties. The amount of leeway (how much "into the future" or "into
```

Comment:
It is unclear what "consistency properties" you mean here.

Reply:
I removed it.